gameFunctions.py

```diff
@@ -85,4 +82,34 @@ def end_the_game(screen, gameScore):
                 if e.key == K_SPACE:
                     return 0
                 elif e.key == K_ESCAPE:
-                    return 1
\ No newline at end of file
+                    return 1
+                
+
+
+def get_high_score():
+    high_score = 0
+    high_score_file = open("highscore","r+")
+    try:
+        high_score = int(high_score_file.read())
+    except IOError:
+        print("There is no highscore yet")
+    except ValueError:
+        print("Can't understand")
+    finally :
+        high_score_file.close()
+        return(high_score)
+
+
```

Comment:
Two newlines between functions should be enough :)
Use pep8 or any other python linter to solve the formatting issues.